models/product-type/src/product-type-draft/presets/sample-data-fashion/accessories.spec.ts

```diff
@@ -0,0 +1,219 @@
+import type {
+  TProductTypeDraft,
+  TProductTypeDraftGraphql,
+} from '../../../types';
+import accessories from './accessories';
+
+describe(`with accessories preset`, () => {
+  it(`should create an accesories product type draft`, () => {
+    const accesoriesProductType = accessories().build<TProductTypeDraft>();
+    expect(accesoriesProductType).toEqual(
```

Comment:
This is a nit, but any reason why `toMatchInlineSnapshot` wasn't used here? Seems easier to maintain

Reply:
My mind is still blown

Reply:
Gotchu fam